codalab/rest/users.py

```diff
@@ -135,7 +138,7 @@ def delete_user():
 
         groups = local.model.batch_get_groups(owner_id=user_id)
         if groups is not None and len(groups) > 0:
-            group_uuids = [group.uuid for group in groups]
+            group_uuids = [group['uuid'] for group in groups]
```

Comment:
Why this change? Was this a bug earlier?

Reply:
yes, some bug existing due to the object/dict difference